packages/certify/certify.0.3.0/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+maintainer:  ["maintenance@identity-function.com"]
+homepage:     "https://github.com/yomimono/ocaml-certify"
+dev-repo:     "git+https://github.com/yomimono/ocaml-certify.git"
+bug-reports:  "https://github.com/yomimono/ocaml-certify/issues"
+doc:          "https://yomimono.github.io/ocaml-certify/doc"
+synopsis:     "CLI utilities for simple X509 certificate manipulation"
+authors: [
+  "Mindy Preston"
+]
+tags: ["org:mirage"]
+
+build: [
+  [ "dune" "build" ]
```

Comment:
```suggestion
  [ "dune" "build" "-p" name "-j" jobs ]
```
`dune build` alone is going to compile everything in debug mode (with some -warn-error). For releases, it should always look like this